conf/messages

```diff
@@ -2223,6 +2236,7 @@ acquisitionPriceBought.agent.indirect.title=What was the acquisition amount?
 acquisitionPriceBought.trust.indirect.title=What was the acquisition amount?
 acquisitionPriceBought.capacitor.indirect.title=How much did the person pay for the shares?
 acquisitionPriceBought.personalRep.indirect.title=How much did the person pay for the shares?
+acquisitionPriceBought.personalRep.indirect.title=
```

Comment:
is this supposed to be here?

Reply:
are we missing a `personalRepInPeriodOfAdmin` and `personalRepInPeriodOfAdmin.agent` titles here?